### PR TITLE
typos

### PR DESCRIPTION
**Pull request by brianteeman**

### CODE REVIEW

File: assets/phpstorm_http/admintools.http [hunk 1]
```diff
@@ -64,7 +64,7 @@ Accept: application/vnd.api+json
 User-Agent: FooBar/1.0
 
 ###
-# Step throught the PHP File Change Scanner
+# Step throug the PHP File Change Scanner
```
Comment (nikosdion) [inline assets/phpstorm_http/admintools.http:67]: Shouldn't this be "through" instead?

Comment (brianteeman) [reply assets/phpstorm_http/admintools.http:67]: Yes it should. Multi tasking, fat fingers and inability to concentrate are to blame

Comment (nikosdion) [reply assets/phpstorm_http/admintools.http:67]: No problem. It happens here too. Ping me when you have changed it and I'll merge this :)

**Comment by brianteeman** [comment]
fixed the typo in the typo

**Pull request merged**